src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -343,7 +344,8 @@ function ClimaLand.Canopy.set_canopy_prescribed_field!(
     t,
 ) where {FT}
     (; LAIfunction, SAI, RAI) = component.parameters.ai_parameterization
-    evaluate!(p.canopy.hydraulics.area_index.leaf, LAIfunction, floor(t))
+    t = t isa ITime ? t : floor(t)
```

Comment:
I think you can remove the `floor` here. It's probably there for historical reasons